dlk/python/dlk/templates/include/memdriver.h

```diff
@@ -21,11 +21,55 @@ limitations under the License.
 #include <stdint.h>
 #include <fcntl.h>
 #include <unistd.h>
+#include <memory>
+#include <system_error>
+
+class FDManager {
```

Comment:
For me, this `FDManager` class represents file descriptor itself, rather than a manager. How about renaming this class to just `FD` or `FileDescriptor` or something?

If you are uncomfortable using such a common name, you can also use namespace.

Reply:
OK, fixed.